### ISSUE TITLE
Not receiving desktop notification for the currently active/visited community. I should receive it always when app is not on the foreground IMO.

### ISSUE THREAD

**Issue by slrslr** [oxen-io/session-desktop#2690]
Hi, I am happy with the Session desktop notifications about new messages in communities except I think that i should also be notified about messages in currently visited community (i mean for example in the Conversations list i have clicked Oxen community - that is my currently active one) and then i switch to other application like a web browser and i do not receive notification about new messages added to Oxen community.

* Device: PC
* OS: Debian 11 with KDE desktop
* Session Version or Git commit hash:  1.10.7 latest AppImage

**Comment by KeeJef** [comment]
Yeah i think we should support displaying community notifications to the user even if that community is opened in the app, as long as the app is not visible/non focused.
